oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/ApdexThresholdConfig.java

```diff
@@ -90,7 +89,9 @@ private synchronized void activeSetting(String config) {
             log.debug("Updating using new static config: {}", config);
         }
         rawConfig = config;
-        updateConfig(new StringReader(config));
+        if (config != null) {
+            updateConfig(new StringReader(config));
+        }
```

Comment:
`config == ""` was an signal to "reset" the config, with this new `if` statement, the config might never been reset even the dynamic config is deleted?

Reply:
fixed